src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticStartAnalysisScope.cs

```diff
@@ -329,10 +320,9 @@ public override void RegisterSyntaxNodeAction(Action<SyntaxNodeAnalysisContext>
     /// </summary>
     internal sealed class AnalyzerOperationBlockStartAnalysisContext : OperationBlockStartAnalysisContext
     {
-        private readonly DiagnosticAnalyzer _analyzer;
         private readonly HostOperationBlockStartAnalysisScope _scope;
 
-        internal AnalyzerOperationBlockStartAnalysisContext(DiagnosticAnalyzer analyzer,
+        internal AnalyzerOperationBlockStartAnalysisContext(
                                                             HostOperationBlockStartAnalysisScope scope,
```

Comment:
Nit: a bit weird formatting (applies to other similar places)


```suggestion
        internal AnalyzerOperationBlockStartAnalysisContext(HostOperationBlockStartAnalysisScope scope,
```

Reply:
➡️ I intentionally left formatting as-is to reduce the diff size. The outcome would have looked normal if the original code had adhered to better formatting practices, but it wasn't my place to alter it.